### PR TITLE
Use Maestro to Update FrameworkUDK 

### PR DESCRIPTION
**Pull request by kythant**
FrameworkUDK Standalone package is here to stay.

To allow Maestro to make automated updates to the FrameworkUDK version, a barebones csproj with PackageDownload on the FrameworkUDK package will be referenced by solutions where the FrameworkUDK version is picked up from the eng folder. Maestro is limited to making changes to version.details.xml or version.props so a package.config cannot be used to do the restore.

### CODE REVIEW

File: build/build-mrt.yml [hunk 1]
```diff
@@ -119,6 +119,13 @@ steps:
         }
       }
 
+- task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+  displayName: 'NuGet restore MrtCore'
+  inputs:
+    restoreSolution: '${{ parameters.MRTSourcesDirectory }}\mrt\MrtCore.sln'
+    feedsToUse: config
+    nugetConfigPath:  nuget.config
```
Comment (huichen123) [inline build/build-mrt.yml:127]: is this needed?

File: dev/MRTCore/mrt/MrtCore.sln [hunk 2]
```diff
@@ -21,18 +21,24 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Windows.Applicati
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MrtCoreUnpackagedTests", "Microsoft.Windows.ApplicationModel.Resources\UnpackagedTests\MrtCoreUnpackagedTests.csproj", "{3C618444-4B80-492E-8972-BFAEF8700A52}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.FrameworkUdk.PackageReference", "..\..\..\eng\PackageReference\FrameworkUdk\Microsoft.FrameworkUdk.PackageReference.csproj", "{4E965899-43FD-4654-9613-15E8EB574E01}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
 		Debug|ARM = Debug|ARM
```
Comment (huichen123) [inline dev/MRTCore/mrt/MrtCore.sln:29]: who builds any cpu? in build pipeline, only x86/x64/arm64 are built

Comment (kythant) [reply dev/MRTCore/mrt/MrtCore.sln:29]: I think this was added when I included the FrameworkUDK csproj

Comment (huichen123) [reply dev/MRTCore/mrt/MrtCore.sln:29]: that's just your config manager setting. but if build pipeline doesn't build anycpu, this is probably not essential.

File: build/build-mrt.yml (hunk 1 above)
Comment (kythant) [reply build/build-mrt.yml:60]: Yes and it replaced all the previous individual restores on each project

**Comment by huichen123** [comment]
>     msbuildArguments: '/restore /binaryLogger:$(Build.SourcesDirectory)/mrtcore.$(buildPlatform).$(buildConfiguration).binlog'

no longer needed

---
Refers to: build/build-mrt.yml:70 in eea6560. [](commit_id = eea6560a9c505322bc046b6ea7739f6e0506a98e, deletion_comment = False)

File: eng/PackageReference/FrameworkUdk/Microsoft.FrameworkUdk.PackageReference.csproj [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- This csproj is to facilitate restoring dependencies from Maestro. These versions are pulled from eng/version.details.xml -->
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
+    <Platforms>x64;x86</Platforms>
```
[the remark this answers is not in the document]
Comment (kythant) [reply eng/PackageReference/FrameworkUdk/Microsoft.FrameworkUdk.PackageReference.csproj:6]: arm64 is not in and it works without it dev\MRTCore\mrt\Microsoft.Windows.ApplicationModel.Resources\projection\Microsoft.Windows.ApplicationModel.Resources.Projection.csproj

**Review by huichen123 — APPROVED** [review]

**Pull request merged**